lib/nightmare.js

```diff
@@ -91,6 +91,22 @@ function Nightmare(options) {
     debug('electron child process exited with code ' + code + ': ' + help[code]);
   });
 
+  Object.defineProperty(this, 'engineVersions', {
+    get: function() {
+      delete this.engineVersions; 
+      return this.engineVersions = JSON.parse(
```

Comment:
> But the assignment ought to work with configurable: true, yes?

Nope. `configurable` is just about things that change the existence of the property or its metadata (i.e. the other property descriptor fields). For the assignment operator to work, `writable` has to be true.